lib/request.js

```diff
@@ -31,6 +31,12 @@ function buildRequest (opts) {
       'http:': new http.Agent(agentOption(opts)),
       'https:': new https.Agent(agentOption(opts))
     }
+    // if (opts.certificate) {
```

Comment:
Why is this commented out?

Reply:
Hello @jsumners , 

The check is done directly in the function `agentOption` directly.

Reply:
If the code path is not useful, remove it.

Reply:
if you mean `opts.certificate....` I use it in the `handleHttp1Req` function. I prefer to check if `opt.certificate` exist rather than both keys `opts.cert` and `opts.key`.